### ISSUE TITLE
[Feature request] Improve "Set background image" 

### ISSUE THREAD

**Issue by stefonarch** [lxqt/lxqt-panel#1970]
**Is your feature request related to a problem? Please describe.**

Atm a background image has to be cut more or less to match the panel size. 
The dialog opens "picture" folder by default.

**Describe the solution you'd like**

* Add a "fit image to panel size" option or making this the default
* Add a collection of suitable background images (gradients and patterns) to `/usr/share/lxqt/panelbackgrounds/`
* Change the dialog to open this directory.

![screen_area_ven_11:14:40_](https://github.com/lxqt/lxqt-panel/assets/10681413/7517420b-4a51-4d18-878c-a1dd3d47541b)

![screen_area_ven_11:15:16_](https://github.com/lxqt/lxqt-panel/assets/10681413/0ca64b5e-6dd3-408f-8723-e7826f6e3450)

Related https://github.com/lxqt/lxqt-panel/issues/1805

**Comment by tsujan** [comment]
> Add a "fit image to panel size" option or making this the default

Yes, that needs to be added. Also https://github.com/lxqt/lxqt-panel/issues/1805 should be fixed.

> Add a collection of suitable background images... Change the dialog to open this directory.

I don't think we need to open the dialog in a specific folder. This isn't about icons.

**Comment by stefonarch** [comment]
> I don't think we need to open the dialog in a specific folder. This isn't about icons.

The main idea us to offer some suiting background images, therefor the specific folder. Average users won't crop images and so on.

**Comment by tsujan** [comment]
I understood your point, but since themes can come with their own background gradients, and because other backgrounds may cause visual problems with them, I think it's better to leave these things to the user.

BTW, perhaps making the bg fit the panel by default, instead of adding an option, is a better approach. Who wants a cut or partial bg?
